clippy.toml

```diff
@@ -0,0 +1 @@
+too-many-arguments-threshold = 8
```

Comment:
Required for current `do_init` / `do_handle` signatures with 8 argument. Default threshold is 7.